views/templates/front/payment.tpl

```diff
@@ -28,6 +28,12 @@
             <div class="bnomics-spinner"></div>
         </div>
 
+        <!-- Display Error -->
+        <div class="bnomics-display-error">
+            <h2>{l s='Display Error'  mod='blockonomics'}{$id_order}</h2>
```

Comment:
We can escape all outputs to avoid security warnings in the prestashop validator
![PrestaShop-Validator (1)](https://github.com/blockonomics/prestashop-plugin/assets/39666372/ddbada20-b2de-4126-8c23-c8795538df66)
`{$id_order|escape:'htmlall'}`

Reply:
Alright.